### PR TITLE
Cleanup playbook configs

### PR DESCRIPTION
**Pull request by xjunior**
Decouples the pb kit helpers needed for the library from the documentation app

#### Screens

No UI change

#### Breaking Changes

Removes configuration files that are not used by playbook itself, but are being shipped and loaded by host apps.

#### Runway Ticket URL

There's no Runway ticket, this work tracks back to the proposition of [RFC 0056](https://github.com/powerhome/rfcs/blob/master/0056-playbook-project-organization.md).

#### How to test this

A smoke test should be enough. Should also smoke test nitro / dev docs.

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **URGENCY** Please select a release milestone
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **SPECS** Please cover your changes with specs.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*

### CODE REVIEW

**Comment by jasoncypret** [comment]
I cut an alpha to test this in Nitro

**Comment by xjunior** [comment]
@jasoncypret this change in specific allows you to just point nitro to playbook and run, without cutting a release:

In Nitro's Gemfile:
```
gem "playbook_ui", github: "powerhome/playbook", branch: "cleanup-plybook-configs"
```

File: playbook/spec/dummy/config/initializers/content_security_policy.rb [hunk 1]
```diff
@@ -1,25 +1,17 @@
...
+  p.script_src  :self, :https, :unsafe_inline
+  p.style_src   :self, :https, :unsafe_inline
 
-# Define an application-wide content security policy
-# For further information see the following documentation
-# https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
+  # To allow connections to the webpack-dev-server running in
+  # a separate docker container
+  if Rails.env.development?
+    p.connect_src :self, :https, 'http://localhost:3040', 'ws://localhost:3040', 'http://0.0.0.0:3040', 'ws://0.0.0.0:3040'
```
Comment (thestephenmarshall) [inline playbook/spec/dummy/config/initializers/content_security_policy.rb:12]: Does this mean we might have HMR now?

**Review by thestephenmarshall — APPROVED** [review]
Looks good so far. I'm a little concerned about changes to `playbook/spec/dummy/config/environments/production.rb` and how they might affect production, but it's kinda hard to know anything until we deploy. Overall, it looks like we opened things up a little in this PR.

**Comment by xjunior** [comment]
@thestephenmarshall the changes to `playbook/spec/dummy/config/environments/production.rb` contain the same thing that was previously shipped in `playbook/config/enbironments/production.rb`, and therefore they were also loaded in production. In other words, nothing changes. Also, this configuration is effective in the demo environment deployed by milano.

File: playbook/spec/dummy/config/initializers/content_security_policy.rb (hunk 1 above)
Comment (xjunior) [reply playbook/spec/dummy/config/initializers/content_security_policy.rb:12]: This is the same thing we already had before, so if it didn't work before, this won't make it better, unfortunately. But it's good to know that we don't currently have it and that we'd like to. I'll try to make it happen when I separate the lib from the website.

**Pull request merged**